code/__DEFINES/mobs.dm

```diff
@@ -284,6 +284,27 @@
 #define	REAGENTS_METABOLISM 0.4	//How many units of reagent are consumed per tick, by default.
 #define REAGENTS_EFFECT_MULTIPLIER (REAGENTS_METABOLISM / 0.4)	// By defining the effect multiplier this way, it'll exactly adjust all effects according to how they originally were with the 0.4 metabolism
 
+// Skills
+#define SKILL_LEVEL_NONE 0
+#define SKILL_LEVEL_NOVICE 1
+#define SKILL_LEVEL_APPRENTICE 2
+#define SKILL_LEVEL_JOURNEYMAN 3
+#define SKILL_LEVEL_EXPERT 4
+#define SKILL_LEVEL_MASTER 5
+#define SKILL_LEVEL_LEGENDARY 6
+
+#define SKILL_MINING "Mining"
```

Comment:
Skip the define and make this a typepath for some future proofing. It makes it easier to put some data in there if we need it. Don't bother instancing them though